src/main/drivers/pwm_mapping.c

```diff
@@ -129,10 +129,16 @@ pwmOutputConfiguration_t *pwmInit(drv_pwm_config_t *init)
 #endif
 
 #if defined(STM32F303xC) && defined(USE_UART3)
-        // skip UART3 ports (PB10/PB11)
+#if defined(AVOID_UART3_FOR_PWM_PPM)
+        if (!init->usePPM)
+            // skip UART3 ports (PB10/PB11)
+            if (init->useUART3 && (timerHardwarePtr->tag == IO_TAG(UART3_TX_PIN) || timerHardwarePtr->tag == IO_TAG(UART3_RX_PIN)))
```

Comment:
Having the UART2 port not configured was part of the test fom the very beginning. With latest code the UART2 even disappears from the ports page when PPM or PWM RX is enabled. In the CLI resource dump everthing looks good as it shold be?

The code from this PR will not be needed since the serial RX always works and the timer for PPM did not get initialized anyhow.